testutil/src/main/proto/spine/protodata/test/doctor.proto

```diff
@@ -38,5 +59,28 @@ enum Incarnation {
     CURRENT = 2;
 }
 
+message Yes {
+}
+
+message Episode {
+
+    uint32 series = 1;
+
+    uint32 episode = 2;
+}
+
+service DoctorsPhone {
+
+    rpc who(google.protobuf.Empty) returns (google.protobuf.Empty) {
+        option idempotency_level = NO_SIDE_EFFECTS;
+    };
+
+    rpc rescue_call(stream spine.people.PersonName) returns (Yes);
+
+    rpc where_are_you(google.protobuf.Empty) returns (stream google.protobuf.StringValue);
+
+    rpc which_actor(stream Episode) returns (stream spine.people.PersonName);
+}
+
 message _DeleteMe {
```

Comment:
Please have a comment on this, since someone _is_ going to delete it eventually.